### DISCUSSION TITLE
can i do inline maths?

### DISCUSSION THREAD

**Discussion by roker15** [react-page/react-page#1079] [Q&A]
can i write inline maths in this editor? or can i copy paste ms word maths formulae in this editor?

**Comment by macrozone** [comment]
you can use this example: https://github.com/react-page/react-page/blob/master/examples/plugins/katexSlatePlugin.tsx

<img width="318" alt="Bildschirmfoto 2021-10-13 um 09 35 25" src="https://user-images.githubusercontent.com/1972353/137088046-88333cda-8729-416a-8e80-8fbf3fdf9418.png">
